Signal/src/call/CallService.swift

```diff
@@ -144,8 +144,10 @@ protocol CallServiceObserver: class {
                 if let oldValue = oldValue {
                     DeviceSleepManager.sharedInstance.removeBlock(blockObject: oldValue)
                 }
+                stopAnyActiveCallTimer()
                 if let call = call {
                     DeviceSleepManager.sharedInstance.addBlock(blockObject: call)
+                    self.startActiveCallTimer()
```

Comment:
Sounds reasonable.  We might want to rename some of the associated methods to give a hint as to their new roles.

maybe: 
- `startCallTimer` 
- `stopAnyCallTimer`
- `ensureUndisplayedCallTerminated`